cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -299,6 +300,14 @@ def publication_additional(self, key, value):
             temp_info.update(pages)
         rel_recid = clean_val("b", v, str)
         if rel_recid:
+            _related.append(
+                {
+                    "related_recid": rel_recid,
+                    "relation_type": OTHER_RELATION.name,
+                    "relation_description": "chapter of"
+                }
+            )
+            _migration.update({"related": _related, "has_related": True})
             _migration["journal_record_legacy_recid"] = rel_recid
```

Comment:
Should we delete the code below or we keep it? ping @kprzerwa  
```diff
-_migration["journal_record_legacy_recid"] = rel_recid
-_migration["has_journal"] = True
-# assume that if we have a parent journal
-# then the doc is a periodical issue
-self["document_type"] = "PERIODICAL_ISSUE"
```

Reply:
In my [pr](https://github.com/CERNDocumentServer/cds-ils/pull/290/files) I removed it because I am implementing it for rule 773

Reply:
Ok, thanks!